pandas/core/groupby.py

```diff
@@ -162,6 +162,17 @@
     a    5
     b    2
     dtype: int64
+
+    When GroupBy object has group_keys=False changes are made to
```

Comment:
I'm not sure what this means. Perhaps you could show the same example with and without `group_keys=False`? Or show an example where changing `group_keys` affects the output? It seems to be identical for both.

Reply:
@TomAugspurger : Indeed, the difference is not as evident as this new doc seems to imply:

https://github.com/pandas-dev/pandas/blob/486bfe8267cd8b71fb322f3b02db3921b0a4e14e/pandas/core/groupby/groupby.py#L1124-L1145

I'm uncertain as to whether this addition really clarifies anything...